content/_index.html

```diff
@@ -36,23 +36,19 @@ <h2>Why Use Epinio ?</h2>
   <div class="grid-four">
     <div>
     <h5>Built for DevOps</h5>
-    <p>Epinio installs into any Kubernetes cluster and brings your
-application from source code to deployment.</p>
+    <p>Epinio installs into any Kubernetes cluster to bring your application from source code to deployment and allow for Developers and Operators to work better together!</p>
     </div>
     <div>
     <h5>Developer Happiness</h5>
-    <p>Focus on your application instead of Container or Kubernetes
-details.</p>
+    <p>Reduce the stress and focus on your application logic instead of Container or Kubernetes details.</p>
     </div>
     <div>
     <h5>Fast &amp; Lightweight</h5>
-    <p>Epinio runs well on your local development machine with k3s
-or minikube.</p>
+    <p>Epinio runs well on your local development machine with <a href="https://rancherdesktop.io">Rancher Desktop</a>, minikube, or k3d.</p>
```

Comment:
Well, minikube still has troubles (#752, #790) on my system

Reply:
How about this language?

Reply:
Since GH didn't show the new diff automatically: `Epinio runs well on your local development machine with <a href="https://rancherdesktop.io">Rancher Desktop</a>, k3d, and many other Kubernetes options.`

Reply:
But we can mention Minikube once we are more confident with it.